<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_source">
        <property name="text">
         <string>Source</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_path_source"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="button_browse_source">
        <property name="text">
         <string>browse</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_path_target"/>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="button_browse_target">
        <property name="text">
         <string>browse</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="button_quit">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_target">
        <property name="text">
         <string>Target</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="button_import">
        <property name="text">
         <string>import</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_includeSubDir">
          <property name="text">
           <string>Include sub folders</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_dirAsDate">
          <property name="text">
           <string>Import pics to different folder as date</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radioBtn_dirAsY">
            <property name="text">
             <string>Year/</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioBtn_dirAsYM">
            <property name="text">
             <string>Year/Month/</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioBtn_dirAsYMD">
            <property name="text">
             <string>Year/Month/Day/</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Menu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSource"/>
    <addaction name="actionTarget"/>
    <addaction name="actionImport"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <addaction name="menu_Menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>about</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionSource">
   <property name="text">
    <string>Source</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionTarget">
   <property name="text">
    <string>Target</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>about QT</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_path_source</tabstop>
  <tabstop>lineEdit_path_target</tabstop>
  <tabstop>button_browse_source</tabstop>
  <tabstop>button_browse_target</tabstop>
  <tabstop>button_import</tabstop>
  <tabstop>button_quit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
